client_code/navigation.py

```diff
@@ -47,9 +47,9 @@ def set_title(text):
     _title_label.text = text
 
 
-def open_form(form_name, full_width=False):
+def open_form(form_name, full_width=False, *args, **kwargs):
```

Comment:
Would it be breaking to make full_width a keyword only argument? If a user did want to make use of args they'd probably get hit with the first arg being used for full_width.